src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationAttribute.kt

```diff
@@ -92,7 +92,7 @@ class LocationAttribute private constructor(
   fun isLocationUsage(usage: LocationUsage) = locationUsage == usage
 
   fun addSchedule(
-    usage: LocationUsage,
+    usage: LocationScheduleUsage,
```

Comment:
Could possibly just be called ScheduleUsage?

Reply:
Maybe as part of the name refactor after this?